source/_components/lightwaverf_energy.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: page
+title: "Lightwave Energy Monitor"
+description: "Instructions on how to integrate Lightwave Energy monitor with Home Assistant"
+date: 2019-08-04 22:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: lightwave.png
+ha_category:
+  - Energy
+ha_release: "0.95.4"
+ha_iot_class: Local Polling
+redirect_from:
+  - /components/lightwave_energy/
+---
+
+[Lightwave Energy](https://lightwaverf.com/products/jsjslw600-lightwaverf-electricity-monitor-and-energy-monitor) integration for Home Assistant allows you to connect Lightwave Energy monitor to Home Assistant
+
+This solution generates two sensors within Home Assistant, one which displays the current usage (in W) and the other is the total amount of electricity used today (in kWh)
+
+
+```yaml
+# Example configuration.yaml entry
+
+sensor lightwave_energy:
```

Comment:
I think that this is not right maybe `sensor:`? And we would like to see only required items in the first example.

Reply:
Its like this because I ported this from the old style components to new and given I have only one sensor class I dont have anything in my __init__.py but all in the sensor.py..

I'll look to migrating it to the new model.

Reply:
Please use the new model